packages/next/src/views/Root/getCustomViewByRoute.ts

```diff
@@ -22,7 +22,7 @@ export const getCustomViewByRoute = ({
     routes: { admin: adminRoute },
   } = config
 
-  const currentRoute = currentRouteWithAdmin.replace(adminRoute, '')
+  const currentRoute = ["/"].includes(adminRoute) ? currentRouteWithAdmin : currentRouteWithAdmin.replace(adminRoute, '')
```

Comment:
This line can likely be completely omitted. The `currentRouteWithAdmin` arg already handles replacing the admin route.

```suggestion
```